### PR TITLE
Kristina button styles

### PR DESCRIPTION
**Pull request by rkcrisafi**
## Ticket
[Notion](https://www.notion.so/teamshares/Buttons-Code-component-ViewComponent-5f3bd911fce147038fac14d1e63d3cfb)

## Description
Add new button classes

### CODE REVIEW

**Review by lizziekoshelev — APPROVED** [review]
A little hard to tell what's happening in git - just confirming all we did was wrap ts-button in not view component and put our own styles at the bottom? If that's the case LGTM

File: scss/_teamshares.scss
Comment (kdonovan) [inline scss/_teamshares.scss:1155]: Super minor, but seems like they all have `font-semibold` -- can we extract that to the top-level `.button-view-component`?

File: scss/_teamshares.scss
Comment (kdonovan) [inline scss/_teamshares.scss:220]: Question on how the design group is approaching styles like these -- it COULD be rewritten to use tailwind classes, which would then make it clear which styles are custom overrides, but I'm not sure which approach is preferable.

e.g. these three lines _could_ be (not sure if they _should_ be) replaced with `@apply flex justify-center items-center`

**Review by kdonovan — APPROVED** [review]
~Would also wait on @lizziekoshelev 's approval as she's taking a closer look at the actual style changes, but~ I now see she's already approved 😅, so LGTM! 🚗

File: scss/_teamshares.scss
Comment (lizziekoshelev) [reply scss/_teamshares.scss:220]: We're going to end up deprecating all the existing styles in this `button,
.ts-button` styling chunk as we create link/button view components, the only changes made to this styling was to declare that existing button styles should not be applied to elements with our new view component classes below

**Pull request merged**
